### ISSUE TITLE
[Simulating matches using a Poisson model]

### ISSUE THREAD

**Issue by yashkarle** [yashkarle/data-science-notes#2]
## TL;DR
List of ideas, articles and code snippets to understand the use of a Poisson model to predict goals and matches

### Blog Posts
[Poisson distribution R tutorial](https://bookdown.org/gabriel_butler/ECON41Labs/tutorial-5-the-poisson-distribution.html#predicting-the-number-of-babies-born-in-a-hospital)
[Danny Page expected goals simulator](https://danny.page/expected_goals.html)

### Project Idea
- Construct a Poisson model from "number of shots + xG of those shots" predicting "goals" for a given team
- a) Goal simulation (xG, xGA, shots_for, shots_faced_against; clean-sheet probabilities can be included here too) 
- b) Match simulation (win_t1/draw/win_t2 probabilities; worth considering opposition strength (FDR from FPL))
- c) Entire season simulation (over all 38 league matches; possibly take into account form over last n matches)

## Key theoretical takeaways
* Randomness in Football in general
* Poisson counting process, Poisson distribution
* Simulation of PL matches based on shots (quantity and quality)

## Useful Code Snippets
[R code snippets](https://github.com/tuangauss/DataScienceProjects/tree/master/R/EPL/prediction)
```R
h_scored = rpois(1, 1/2 * (t_ave_h_s + t_ave_a_c))
a_scored = rpois(1, 1/2 * (t_ave_a_s + t_ave_h_c))
```
